esp-alloc/CHANGELOG.md

```diff
@@ -5,20 +5,26 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## Unreleased
 
 ### Added
 
-- `esp_alloc::heap_allocator!` now accepts attributes, e.g., `esp_alloc::heap_allocator!(#[link_section = ".dram2_uninit"] size: 64000)` (#3133)
-
 ### Changed
 
-- `esp_alloc::heap_allocator!` syntax has been changed to `esp_alloc::heap_allocator!(size: 64000)` (#3135)
-
 ### Fixed
 
 ### Removed
 
+## [0.7.0] - 2025-02-24
```

Comment:
Since we will start tagging all the crates, I guess we should add a link to `esp-alloc-v0.7.0` tag? Same applies for all the other crates